src/Lesson4/Human.java

```diff
@@ -0,0 +1,16 @@
+package Lesson4;
+
+public class Human {
+    public int weight;
+    public int age;
+    public String first_name;
+    public String last_name;
+    public int height;
+    public Human() {
+        height = 175;
+        age = 27;
+        first_name = "Ilon";
+        last_name = "Mask";
+        weight = 75;
+    }
```

Comment:
There is one more task:
b) Add to the class a constructor with parameters which initialize all the fields.